jbpm-workitems/jbpm-workitems-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -3,74 +3,87 @@
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
     xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>org.jbpm.contrib</groupId>
-    <artifactId>workitems</artifactId>
-    <version>${archetypeVersion}</version>
-  </parent>
 
-  <artifactId>${rootArtifactId}</artifactId>
-  <name>${rootArtifactId}</name>
+  <groupId>${groupId}</groupId>
+  <artifactId>${artifactId}</artifactId>
+  <name>${artifactId}</name>
+  <version>${version}</version>
   <description>Workitem Desciption</description>
   <organization>
     <name>organization</name>
   </organization>
-
+  <properties>
```

Comment:
IMO we should rely on a BOM here for the dependency version management, so that users end up with our well-known combination of dependencies.

Reply:
Do you mean like this https://github.com/kiegroup/jbpm/pull/1652/files#diff-2783c2b62f84d175147d71379e38a18eR8? 
if the service task has the parent pom, it can't upload into BC. You can see comments of https://issues.redhat.com/browse/RHPAM-2889

Reply:
if this archetype is strictly for jbpm it should use jbpm-bom [[1](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/jbpm-bom/pom.xml)]

Reply:
if use this one, all service tasks will have parent pom(boss.releases.repo), let service task can not upload into BC.

Reply:
jbpm-pom have a parent pom(jboss-parent),if the archetype dependent on it, It It will cause ssues.redhat.com/browse/RHPAM-2889. so I create a new archetype to avoid it.

Reply:
The problem was the distributionManagement definition in the jboss-parent, right? If you use the BOM just for importing in the dependencyManagement section the issue should not appear IMO.

Reply:
you can define empty distributionManagement there or avoid it completely but that will require more content in pom.xml of the generated project by archetype

Reply:
I don't mind having this parent-less archetype to be honest, just the version management should rely on BOMs.

Reply:
> you can define empty distributionManagement there or avoid it completely but that will require more content in pom.xml of the generated project by archetype

I think maybe use empty distributionManagement will work, I will take a try.

Reply:
> you can define empty distributionManagement there or avoid it completely but that will require more content in pom.xml of the generated project by archetype

@mareknovotny I define empty distributionManagement, but it not work.

[1] will get all parent pom's distributionManagement.

@jstastny-cz @mareknovotny In the PR, It has two archetype,jbpm-workitems-repository-archetype rely on kie-parent. This Archetype allows you to easily start building a new jBPM Workitem and easily be contributed to the workitem repository.
another is jbpm-workitems-archetype,it is not related to kie-parent. It just allows users to easily create new workItem.

You can see Tihomir's comment on the issue[2].

[1] https://github.com/kiegroup/appformer/blob/dfff1ac7ed5bc865dd442c71fa6a403aecef2061/uberfire-m2repo-editor/uberfire-m2repo-editor-backend/src/main/java/org/guvnor/m2repo/backend/server/repositories/DistributionManagementArtifactRepository.java#L149
[2] https://issues.redhat.com/browse/JBPM-8995